homeassistant/components/sense/binary_sensor.py

```diff
@@ -66,6 +67,14 @@ def __init__(
         self._attr_unique_id = f"{sense_monitor_id}-{self._id}"
         self._attr_icon = sense_to_mdi(device.icon)
         self._device = device
+        self._attr_device_info = DeviceInfo(
+            name=f"Sense {sense_monitor_id} - {device.name}",
```

Comment:
I think we should name this device the device name as that is user recognisable. They can view to which monitor it belongs via the device info page

Reply:
Also because i think it's a good idea that we migrate to has entity name = True, where all entity names are appended to the device name